index.js

```diff
@@ -40,7 +40,11 @@ function fastifyMultipart (fastify, options, done) {
 
     const req = this.req
 
-    const stream = new Busboy(Object.assign({ headers: req.headers }, options))
+    const busboyOptions = { headers: req.headers }
+    for (let opt of Object.keys(options)) {
+      busboyOptions[opt] = options[opt]
+    }
```

Comment:
`for of` is super slow as well (also using `let` in loops is slow), you'd need a nice old `for (var i = 0; i< n; i++)`.

Reply:
Do you mean something like this? 

```js
    const busboyOptions = { headers: req.headers }
    const keys = Object.keys(options)
    for (var i = 0; i < keys.length; i++) {
      busboyOptions[keys[i]] = options[keys[i]]
    }
    const stream = new Busboy(busboyOptions)
```